packages/server/src/app.module.ts

```diff
@@ -25,10 +27,14 @@ import { UsersModule } from './users/users.module';
         migrationsRun: true,
       }),
     }),
+    ThrottlerModule.forRoot({
```

Comment:
Does this apply to all routes or just `/`? Can a route define its own rate limit? Say, 300 per second for GETs (since those are low effort and quite common) vs 60 for PUT/POST/PATCH/DELETE (since those are more work)?

Reply:
It'll apply to any route decorated with `@UseGuards(ThrottlerGuard)`. We can override it for each individual route as needed.